src/main/java/com/example/tweet/common/errors/NumberOfAffectedRecordError.java

```diff
@@ -0,0 +1,7 @@
+package com.example.tweet.common.errors;
+
+public class NumberOfAffectedRecordError extends Exception {
```

Comment:
```suggestion
public class DBUpdateException extends Exception {
```
このくらいの方が直感的でわかりやすい気がしました。
※現状でも命名で言わんとしていることは分かります。

Reply:
例外クラスの命名を変更しました。
d70ca9ca6d635da5480b3f360307336aa2676b7c